examples/school/views/SchoolRemove.js

```diff
@@ -0,0 +1,65 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { graphql, compose } from 'react-apollo';
+
+import Formsy from 'formsy-react';
+import Select from '../components/Select';
+import { removeSchool } from '../graphql/schools';
```

Comment:
maybe change removeSchool for only remove, the context is in the import '../graphql/**schools**'